src/main/io/vtx_control.h

```diff
@@ -28,7 +28,9 @@
 #include "pg/pg.h"
 #include "fc/rc_modes.h"
 
-#define MAX_CHANNEL_ACTIVATION_CONDITION_COUNT  10
+#ifndef MAX_VTX_CONDITIONS
+#define MAX_VTX_CONDITIONS  10
+#endif
```

Comment:
Perhaps we add a sensible max, not sure how much

```suggestion
#endif

STATIC_ASSERT(MAX_CHANNEL_ACTIVATION_CONDITION_COUNT <= 99, "MAX_CHANNEL_ACTIVATION_CONDITION_COUNT must be <= 99");
```